lib/katello/plugin.rb

```diff
@@ -218,8 +218,9 @@
        :parent => :lab_features_menu,
        :turbolinks => false
 
-  allowed_template_helpers :subscription_manager_configuration_url, :repository_url
+  allowed_template_helpers :subscription_manager_configuration_url, :repository_url, :sortable_version
```

Comment:
No need to add here, this is handled by `extend_template_helpers` (in fact, the other two are also redundant)

Reply:
Is it safe to remove the whole line? We will be backporting this one.

Reply:
it is, but it can be done in a separate commit:

https://github.com/theforeman/foreman/blob/develop/app/registries/foreman/plugin.rb#L578